src/bloqade/task/quera.py

```diff
@@ -73,7 +115,15 @@ def fetch(self) -> "QuEraTask":
         return self
 
     def pull(self) -> "QuEraTask":
-        # blocking, force pulling, even its completed
+        """
+        Forcefully pulls the task results from the backend.
```

Comment:
```suggestion
        Blocks execution until task is in a stopped state, e.g. Completed, Cancelled, Failed, etc.
```

Wrong behavior, this function is used to Poll the remote API until the task is completed.